clientgui/ProjectInfoPage.cpp

```diff
@@ -42,10 +42,14 @@
 #include "res/windowsicon.xpm"
 #include "res/macosicon.xpm"
 #include "res/linuxicon.xpm"
+#include "res/androidicon.xpm"
 #include "res/freebsdicon.xpm"
+#include "res/linuxarmicon2.xpm"
```

Comment:
Why `2` in this and others?

Reply:
To clearly differentiate my second generation icons from the previous set.

Reply:
Ok, thanks for explaining.